### PR TITLE
Update master version to R2021b revision 1

### PR DESCRIPTION
**Pull request by stefaniapedrazzi**
It is not planned to release R2021b revision 1 but to jump directly to R2021c.
However, I think it is better to set it to R2021b revision 1 until R2021c is ready so that it will be easier to release a bug fix version if we should have a urgent need for it.

### CODE REVIEW

File: scripts/new_version/new_version.sh [hunk 1]
```diff
@@ -81,5 +79,7 @@ then
 
   $CURRENT_DIR/new_version_file.sh "wwi\/$old_version_without_revision\/" "wwi\/$new_version_without_revision\/" $WEBOTS_HOME/docs/dependencies.txt
 
+  $CURRENT_DIR/new_version_file.sh $old_version_without_revision $new_version_without_revision $WEBOTS_HOME/projects/humans/c3d/plugins/robot_windows/c3d_viewer_window/c3d_viewer_window.html
```
Comment (lukicdarkoo) [inline scripts/new_version/new_version.sh:82]: This seems to be an unrelated change, why is it changed?

Comment (stefaniapedrazzi) [reply scripts/new_version/new_version.sh:82]: It is better to change the version of files used in the c3d robot window only on major released.
So it is related because if run the script without fixing this the c3d robot window file will get updated.
